bokehjs/src/coffee/models/plots/plot.ts

```diff
@@ -160,6 +161,8 @@ export interface Plot extends Plot.Attrs {}
 
 export class Plot extends LayoutDOM {
 
+  reset = new Signal0(this, "reset")
```

Comment:
Only declaration can be here. Definition must be moved to `initialize()`. This is because `initialize()` doesn't work particularly well with this style of member initialization. This will work by default though (in deferred mode), but not when used directly in bokehjs.

Reply:
Sure, I will finish this. I wasn't consistent myself with this. I will need to review how classes are initialized to make sure that there is one, correct way.